src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 14px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  }
+
+  &__imac_img {
+    display: flex;
+    justify-content: center;
+    margin: 15px 5px 45px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-bottom: 6px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    color: #000;
+  }
+
+  &__code {
+    margin: 0 0 15px;
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 13px;
+    color: #000;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 20px;
+  }
+
+  &__feedback {
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 13px;
+    color: #000;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin-bottom: 20px;
+  }
+
+  &__price_text {
+    margin: 0;
+    font-weight: 400;
+    font-size: 11px;
+    line-height: 15px;
+    color: #000;
+  }
+
+  &__price_amount {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;
+    color: #000;
+  }
+
+  &__button {
+    margin-top: 0;
+    margin-bottom: 0;
+    display: flex;
+    box-sizing: border-box;
```

Comment:
you have already specified this rule in your main.scss

Reply:
agree. box-sizing: border-box is already written for all the elements